### ISSUE TITLE
Is that free, or do we need a subscription from WooCommerce?

### ISSUE THREAD

**Issue by ehtashamGH** [wp-premium/woocommerce-subscriptions#12]
Does using this require a subscription from WooCommerce, or is it available for free to use?

**Comment by gmt-it** [comment]
It's free ;-)

**Comment by ehtashamGH** [comment]
Thank you!
Does this plugin support recurring payments?
My local bank provides APIs for both initial and recurring payments. Is this plugin compatible with these APIs?
If so, could you please outline the steps for integration?

**Comment by ghost** [comment]
How safe is this to use, if compared to the 270$/year official plugin? This gets updates, and security updates and all that? Amzing that this is free :)

**Comment by gmt-it** [comment]
Well, I had the extension as a subscription until an Indian programmer pointed out to me that this one on github is exactly the extension that you pay for at woo. If you don't need woo's support, it saves a lot of money. You get all the updates for the paid version here too (even sooner)!

**Comment by ghost** [comment]
> Well, I had the extension as a subscription until an Indian programmer pointed out to me that this one on github is exactly the extension that you pay for at woo. If you don't need woo's support, it saves a lot of money. You get all the updates for the paid version here too (even sooner)!

Yeah thats what i was looking for and i saw that ppl behind this "fork" of the plugin are themselves developers of other plugins. I was just afraid that this might contain evil code. 

But thanks for clarification, this is excatly what I need insted of paying 280$ a year!!!

**Comment by jesskams** [comment]
Thank you very much for this plugin, just a request. In the original plugin you could choose how many days, months or years could pass between payments, for example: 45 days, if you subscribe on January 1st, the next payment will be on February 14th. Have this options its very helpfull.

**Comment by jjxtra** [comment]
How is this free if woo puts the download of their stuff behind a paywall? Is this forked from a public repo or do you have an agreement with woo, or does their license permit redistribution?

**Comment by remcotolsma** [comment]
https://github.com/pronamic/woocommerce-subscriptions/blob/15b2c611fc27ef8d0ff28c4a816250059361de49/woocommerce-subscriptions.php#L17-L20
